src/main/java/com/github/fishio/EnemyFish.java

```diff
@@ -112,9 +110,9 @@ public void preMove() {
 	 */
 	public void limitVx() {
 		if (vx > 0) {
-			vx = Math.max(EnemyFishFactory.MIN_EFISH_SPEED, Math.min(vx, EnemyFishFactory.MAX_EFISH_SPEED));
```

Comment:
*Auto-generated comment*

    
    The String literal "MIN_EFISH_SPEED" appears 4 times in this file; the first occurrence is on line 113